### PR TITLE
Add client sprite for Bases

### PR DESCRIPTION
**Pull request by konwinkler**
1. temporary developer art sprite asset
2. next step is to load base information from server map

### CODE REVIEW

**Review by Nitewriter — APPROVED** [review]

File: client/src/sprites/baseSprite.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export default class BaseSprite extends AppSprite {
+    static loadAsset(game: Phaser.Game): void {
+        game.load.image('base', 'base.png')
+    }
+
+    constructor(game: Phaser.Game, x: number, y: number) {
+        super(game, x, y, 'base')
+
+        this.width = 80
+        this.height = 120
```
Comment (kconner) [inline client/src/sprites/baseSprite.ts:12]: We will want 240x240

Comment (konwinkler) [reply client/src/sprites/baseSprite.ts:12]: ok, will update!

File: client/src/states/preloader.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import AppState from './appState'
 import HeroSprite from '../sprites/heroSprite'
 import LifeBarSprite from '../sprites/lifeBarSprite'
 import FogSprite from '../sprites/fogSprite'
+import BaseSprite from '../sprites/baseSprite'
 
 export default class Preloader extends AppState {
```
Comment (kconner) [inline client/src/states/preloader.ts:8]: Add a preload call below as BaseSprite.loadAsset() with the others

Comment (konwinkler) [reply client/src/states/preloader.ts:8]: yea, got lost in my merge, will create bugfix PR

**Pull request merged**